scss/misc/_context_menu.scss

```diff
@@ -35,6 +35,7 @@
   .item {
     line-height: var(--local-menu-item-Line-height);
     padding: var(--local-menu-item-vertical-padding) 12px;
+    text-transform: capitalize;
```

Comment:
i think, this can go wrong in many languages, not only german. even in english, words as "and" are not capitalized in menus.  i would remove auto-capitalize.